### DISCUSSION TITLE
umi4 引入react组件报错:Invalid plugin umi-plugin-react,[object Object], it must be string.

### DISCUSSION THREAD

**Discussion by wang-jie-ss** [umijs/umi#8996] [Q&A]
![image](https://user-images.githubusercontent.com/92045183/184921815-10298175-a8c7-47a4-85a4-8ae26463763b.png)

config.js的配置:
export default {
    plugins: [
        ['umi-plugin-react', {
            // 这里暂时还没有添加配置，该插件还不会有作用，我们会在后面的课程按照需求打开相应的配置
            antd: true
        }],
    ],
    routes: [{
        path: '/',
        component: './HelloWorld',
    }],
}

package.json:
{
  "name": "day05",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "devDependencies": {
    "@umijs/max": "^4.0.12",
    "umi-plugin-react": "^1.15.9"
  },
  "dependencies": {
    "antd": "^4.22.5"
  }
}

**Comment by fz6m** [comment]
max 默认内置了 antd ，不需要 `umi-plugin-react` 了。
